testing/mapnik/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Markus Juenemann <markus@juenemann.net>
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=mapnik
+pkgver=v3.0.12
+pkgrel=0
+pkgdesc="An open source toolkit for developing mapping applications"
+url="http://mapnik.org/"
+arch="all"
+license="LPGL2+"
+depends="boost boost-regex freetype zlib harfbuzz postgresql-client sqlite-libs cairo libpng libjpeg tiff libwebp libxml2 proj4 gdal icu"
+depends_dev=""
+makedepends="$depends_dev python scons boost-dev freetype-dev zlib-dev harfbuzz-dev postgresql-dev sqlite-dev cairo-dev libpng-dev tiff-dev libwebp-dev libxml2-dev proj4-dev gdal-dev icu-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://github.com/mapnik/mapnik/releases/download/$pkgver/mapnik-$pkgver.tar.bz2"
+
+builddir="$srcdir/$pkgname-$pkgver"
+build() {
+	cd "$builddir"
+	python scons/scons.py configure PREFIX=/usr 
+	python scons/scons.py           PREFIX=/usr
+}
+
+package() {
+	cd "$builddir"
+	python scons/scons.py install   PREFIX=/usr DESTDIR="$pkgdir"
```

Comment:
For some reason, a rebuild of everything seems to happen here. It double the build time. Any ideas why it might happen?